spec/features/user_can_see_a_book_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+# When I visit a book show page, I see the book title and a list of reviews for that book,
+# including the body and the user that created that review.
+# I also see the rating (1-5; you do not need to validate that this falls within this range)
+# that the user gave the book.
+
+describe "user can see individual book information" do
+  context "visits book show page" do
+    it "displays book informations" do
+      book = Book.create!(title: "Catcher in the Rye")
+
+      visit book_path(book)
+
+      expect(page).to have_content("Title: Catcher in the Rye")
+    end
+
+    it "displays book reviews" do
+      book = Book.create!(title: "Catcher in the Rye")
+      review_1 = book.reviews.create!(user_name: "Billy",
+                                      body: "I enjoyed this one!",
+                                      rating: 5)
+
+      visit book_path(book)
+
+      expect(page).to have_content("Billy")
+      expect(page).to have_content("I enjoyed this one!")
+      expect(page).to have_content(5)
+    end
```

Comment:
I'd rather see these expectations being called on `review_1.body` etc. This saves a reader some time in understanding the context. I don't have to think about what does "Billy" signify.